src/server/modules/user/UserController.ts

```diff
@@ -98,7 +98,7 @@ export class UserController {
         return
       }
       if (error instanceof AlreadyExistsError) {
-        const newErrorMessage = `${error.message} Click here to find out more`
+        const newErrorMessage = `${error.message} Click here to find out who is the link owner & request for ownership.`
```

Comment:
btw sometime ago dat and I were looking through the codebase and saw this line and agreed that the backend shouldn't be sending back this kind of error message lol (why is the backend telling the client to "Click here"?? that should be the frontend's responsibility)
but this change alone is good!